### PR TITLE
docs(multizone): add leader and follower information

### PR DESCRIPTION
**Pull request by slonka**
Can someone help me fill this out? I always struggle with remembering who connects to whom, what runs on a leader, what runs on followers and what runs on both.

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_src/production/deployment/multi-zone.md [hunk 1]
```diff
@@ -69,13 +69,28 @@ A multi-zone deployment includes:
   - Send zone ingresses down to zone control plane.
   - Keep an inventory of all data plane proxies running in all zones (this is only done for observability but is not required for operations).
   - Reject connections from data plane proxies.
+  - Leader
+    - Writes to the store
```
Comment (jakubdyszkiewicz) [inline app/_src/production/deployment/multi-zone.md:73]: But you can also write stuff to a follower, for example PUT request to create a policy can go to any instance

File: app/_src/production/deployment/multi-zone.md [hunk 2]
```diff
@@ -69,13 +69,28 @@ A multi-zone deployment includes:
...
+    - Does insights processing
+    - Other leader components (CP catalog, gc, subscription finalizer, inter cp server)
```
Comment (jakubdyszkiewicz) [inline app/_src/production/deployment/multi-zone.md:75]: I feel like this is might be quite cryptic for the users reading this doc.

Same with "insights processing".

File: app/_src/production/deployment/multi-zone.md [hunk 3]
```diff
@@ -69,13 +69,28 @@ A multi-zone deployment includes:
...
+  - Followers
+    - Run default components
```
Comment (Automaat) [inline app/_src/production/deployment/multi-zone.md:77]: what are `default components` in this context?

**Comment by slonka** [comment]
Will reopen if this is needed again.

**Pull request closed without merging**